src/lib/zkapp.ts

```diff
@@ -107,11 +118,6 @@ function method<T extends SmartContract>(
     target,
     methodName
   );
-  let returnType: Provable<any> = Reflect.getMetadata(
-    'design:returntype',
-    target,
-    methodName
-  );
```

Comment:
we no longer infer the method return type, instead there is `@method.returns(...)`